finish/src/test/java/it/io/openliberty/guides/inventory/SystemResourceIT.java

```diff
@@ -105,27 +101,10 @@ private static boolean isServiceRunning(String host, int port) {
     }
     // end::isServiceRunning[]
 
-    private static String getProtocol() {
-        return System.getProperty("test.protocol", "https");
-    }
-
-    private static boolean testHttps() {
-        return getProtocol().equalsIgnoreCase("https");
-    }
-
     // tag::createRestClient[]
     private static SystemResourceClient createRestClient(String urlPath)
             throws KeyStoreException {
```

Comment:
The line [`builder.trustStore(KeyStore.getInstance("PKCS12"));`](https://github.com/OpenLiberty/draft-guide-testcontainers/blob/draft/finish/src/test/java/it/io/openliberty/guides/inventory/SystemResourceIT.java#L120) in the earlier version before the simplification can throw a `KeyStoreException` if there's an issue creating a KeyStore of type "PKCS12". 

Since it isn't presenting in the new code and that we're simplifying the code to get rid of the SSL setup or keystore stuff, the `throws KeyStoreException` in the signature of the `createRestClient` method is unnecessary and can be safely removed.

```suggestion
```